appveyor.yml

```diff
@@ -300,6 +263,10 @@ for:
           New-Item .\out\Default\gen\node_headers\Release -Type directory
           Copy-Item -path .\out\Default\electron.lib -destination .\out\Default\gen\node_headers\Release\node.lib
       - cd electron
+      - ps: >-
+          if ($env:TARGET_ARCH -eq 'ia32') {
+            $env:npm_config_arch = "ia32"
+          }
```

Comment:
Could we add a small note that this is a workaround to an issue setting that environment variable in the AppVeyor config? Good to note that this shouldn't be necessary, but is for some reason.